src/Microsoft.DotNet.Wpf/src/WindowsBase/MS/Internal/ContentType.cs

```diff
@@ -278,8 +278,8 @@ public override string ToString()
                 if (!_isInitialized)
                     return String.Empty;
 
-                Debug.Assert(String.CompareOrdinal(_type, String.Empty) != 0
-                   || String.CompareOrdinal(_subType, String.Empty) != 0);
+                Debug.Assert(!string.IsNullOrEmpty(_type)
+                             || !string.IsNullOrEmpty(_subType));
```

Comment:
@lindexi, after going through the code, it looks like _type will never be null, so how about using `_type.Length == 0` and `_subType.Length == 0` here ?

What are your thoughts about this ?

Reply:
@dipeshmsft I think the code on debugging can be more flexible. When I wrote this code, I didn’t consider this at the time. Thank you for pointing it out.

Reply:
@dipeshmsft I think that Debug.Assert code is used to intercept unexpected logic during the development process. Since the judgment of the _type field is not null does not occur in the ToString method, in other words, it is possible that developers will break the convention of _type field not being null in subsequent development processes. Therefore, I think that retaining the use of the string.IsNullOrEmpty method can better intercept problems during the development process.